### PR TITLE
Update messages.yml

### PR DESCRIPTION
**Pull request by jojoPL1987**

### CODE REVIEW

File: src/_locales/pl/messages.yml [hunk 1]
```diff
@@ -440,7 +442,7 @@ menuFindScripts:
   message: Znajdź skrypty dla tej strony
 menuMatchedFrameScripts:
   description: Label for menu listing matching scripts in sub-frames.
-  message: ''
+  message: Skrypty tylko w ramkach pomocniczych
```
Comment (gera2ld) [inline src/_locales/pl/messages.yml]: Is the trailing space kept intentionally? If so, quotes are needed.

Comment (jojoPL1987) [reply src/_locales/pl/messages.yml]: Mistake. I corrected it. Excuse me.

PS. When will I see newly added translations in Violentmonkey?

Comment (gera2ld) [reply src/_locales/pl/messages.yml]: In the next release, I guess in a few days.

Comment (jojoPL1987) [reply src/_locales/pl/messages.yml]: OK, thanks for info.

**Pull request merged**